packages/rx_bloc_cli/mason_templates/bricks/feature_counter/__brick__/lib/l10n/network_lookup/counter_lookup.dart

```diff
@@ -0,0 +1,33 @@
+{{> licence.dart}}
```

Comment:
Please confirm this lookup is added conditionally based on the executable input

Reply:
Yes, in fact, it is added conditionally. The `Counter` feature is locked behind the `enable-feature-counter` flag and thus won't generate if not specified. This is for this and the other items mentioned. So this should be good unless I'm mistaken for something else.

Reply:
I see. That's great!